SSO: add small paragraph mentioning how to enable it on commercial (#8063)
readthedocs/readthedocs.org: docs/commercial/single-sign-on.rst

In case you want a user to have access to your documentation project under Read the Docs,
that user just needs to be granted permissions in the VCS repository associated with it.

You can enable this feature in your organization by going to

> "your organization detail's page" > :guilabel:`Settings` > :guilabel:`Authorization`

Reviewer: ```suggestion
your organization's detail page > :guilabel:`Settings` > :guilabel:`Authorization`
```

> and selecting :guilabel:`GitHub, GitLab or Bitbucket` as provider.

Reviewer: If the option is literally "GitHub, GitLab, or Bitbucket", this looks good. If these are separate options in the dropdown, they should be individual guilabel roles

Author: We have only one option in the drop-down for now since it's the same configuration for all of them.

However, you can define which Teams users matching your company's domain email address will auto-join when they Sign Up.
Read the following sections to learn how to grant read and admin access.

You can enable this feature in your organization by going to

> "your organization detail's page" > :guilabel:`Settings` > :guilabel:`Authorization`

Reviewer: ```suggestion
your organization's detail page > :guilabel:`Settings` > :guilabel:`Authorization`
```

> and selecting :guilabel:`Google` as provider and writing your ``company.com`` domain in :guilabel:`Domain` field.

Reviewer: ```suggestion
and selecting :guilabel:`Google` as provider and specifying your Google Workspace domain in the :guilabel:`Domain` field.
```

Author: If we want to use "Google Workspace domain" here we should change this in the rest of the document as well.

Author: Also, we are using GSuite in all the other places, so that's another required change.
